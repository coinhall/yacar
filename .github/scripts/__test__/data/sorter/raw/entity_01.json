[
  {
    "website": "https://t.co/ejYaePobxy",
    "telegram": "https://t.co/kiLJmew5Ls",
    "name": "Astroport",
    "twitter": "https://twitter.com/astroport_fi"
  },
  {
    "coinmarketcap": "https://coinmarketcap.com/currencies/cosmos/",
    "name": "Cosmos",
    "telegram": "https://t.me/cosmosproject",
    "twitter": "https://twitter.com/cosmos",
    "website": "https://cosmos.network/",
    "coingecko": "https://www.coingecko.com/en/coins/cosmos-hub"
  },
  {
    "telegram": "https://t.me/scrtCommunity",
    "name": "Secret Network",
    "coingecko": "https://www.coingecko.com/en/coins/secret",
    "website": "scrt.network",
    "twitter": "https://twitter.com/SecretNetwork",
    "coinmarketcap": "https://coinmarketcap.com/currencies/secret/"
  },
  {
    "name": "Test"
  }
]
