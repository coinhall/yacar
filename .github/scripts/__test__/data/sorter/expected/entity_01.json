[
  {
    "name": "Astroport",
    "website": "https://t.co/ejYaePobxy",
    "telegram": "https://t.co/kiLJmew5Ls",
    "twitter": "https://twitter.com/astroport_fi"
  },
  {
    "name": "Cosmos",
    "website": "https://cosmos.network/",
    "telegram": "https://t.me/cosmosproject",
    "twitter": "https://twitter.com/cosmos",
    "coinmarketcap": "https://coinmarketcap.com/currencies/cosmos/",
    "coingecko": "https://www.coingecko.com/en/coins/cosmos-hub"
  },
  {
    "name": "Secret Network",
    "website": "scrt.network",
    "telegram": "https://t.me/scrtCommunity",
    "twitter": "https://twitter.com/SecretNetwork",
    "coinmarketcap": "https://coinmarketcap.com/currencies/secret/",
    "coingecko": "https://www.coingecko.com/en/coins/secret"
  },
  {
    "name": "Test"
  }
]
